src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizerFactory.cs

```diff
@@ -180,7 +191,8 @@ public IStringLocalizer Create(string baseName, string location)
                 new ResourceManager(baseName, assembly),
                 assembly,
                 baseName,
-                _resourceNamesCache);
+                _resourceNamesCache,
+                _loggerFactory);
```

Comment:
Let's just pass the ILogger in. Doesn't look like the ResourceManagerStringLocalizer would need to spawn different kinds of ILogger instances.